src/get-environments.ts

```diff
@@ -37,15 +37,13 @@ function repositoriesEqual(rep1: string, rep2: string): boolean {
   );
 }
 
-export async function getEnvironmentsForBranch() {
-  const apiKeyCredentials = getApiKeyCredentials();
-
+export async function getEnvironmentsForBranch(api: ApiClient) {
```

Comment:
rename to `apiClient` please (here and the other files that use just `api`)
you use both options, let's keep it the same across the places, and IMO the `apiClient` is the better one

Reply:
d500b72c07620b49b24a0d2dd7026b68669a51f3